dashboard/app/views/levels/_form.html.haml

```diff
@@ -7,11 +7,9 @@
       = f.label :name
       = f.text_field :name, placeholder: 'Level Name'
   / Only enter name to create new records of certain level types.
-  - unless @level.new_record? && (@level.is_a?(Grid) || @level.is_a?(Calc) || @level.is_a?(Eval) || @level.is_a?(Artist) || @level.is_a?(Applab) || @level.is_a?(NetSim))
-    / only Blockly-type levels have Concepts
-    = render partial: 'levels/editors/concepts', locals: {f: f} if @level.is_a? Blockly
-    / Video dropdown not yet implemented for dsl-defined levels
-    = render partial: 'levels/editors/video', locals: {f: f} unless @level.is_a? DSLDefined
+  - unless @level.new_record? && @level.is_a?(Blockly)
```

Comment:
Ah gotcha, that's in the line below (line 11 in the added code).  The `levels/editors/all` partial now contains fields common to all `Level` types (concepts, videos and callouts).  The change on line 10 is just cleanup -- it's there so that users enter just the name when creating a new `Blockly` level, before we show all the fields.

<img src="https://cloud.githubusercontent.com/assets/413693/6975577/c66ea2a2-d952-11e4-9ede-13898cadf975.png" width="250">

I think it's written that way because clicking any of the "edit block" links will navigate away from the page, which will cause problems when the record hasn't been created yet.